npm-shrinkwrap.json

```diff
@@ -4710,7 +4860,7 @@
     },
     "mocha": {
       "version": "3.2.0",
-      "from": "mocha@>=3.0.0 <4.0.0",
+      "from": "https://registry.npmjs.org/mocha/-/mocha-3.2.0.tgz",
```

Comment:
Huh?!? Why has this `from` specifier changed from a version-range to a URL?

Reply:
Weird. Fixed